### PR TITLE
feat: history command

### PR DESCRIPTION
**Pull request by radulucut**
- adds new command `history` (#89)

```bash
globalping history
1 | 2024-03-27 11:56:46 | ping google.com
> https://www.jsdelivr.com/globalping?measurement=itcR65tYCqbouXib
- | 2024-03-27 11:57:01 | dns google.com from last
> https://www.jsdelivr.com/globalping?measurement=kWc5UBK9A6G4RUYM
2 | 2024-03-27 11:57:20 | traceroute google.com from New York --limit 2
> https://www.jsdelivr.com/globalping?measurement=Yz7A1UifUonZsC3C
3 | 2024-03-27 11:57:37 | mtr google.com from New York --limit 2
> https://www.jsdelivr.com/globalping?measurement=SX1NBgfDKiabM1vZ
4 | 2024-03-27 11:57:52 | http google.com from London,Belgium --limit 2 --method get --ci
> https://www.jsdelivr.com/globalping?measurement=eclwFSYX0zgU10Cs

globalping history --first 2
1 | 2024-03-27 11:56:46 | ping google.com
> https://www.jsdelivr.com/globalping?measurement=itcR65tYCqbouXib
- | 2024-03-27 11:57:01 | dns google.com from last
> https://www.jsdelivr.com/globalping?measurement=kWc5UBK9A6G4RUYM

globalping history --last 2
4 | 2024-03-27 11:57:52 | http google.com from London,Belgium --limit 2 --method get --ci
> https://www.jsdelivr.com/globalping?measurement=eclwFSYX0zgU10Cs
3 | 2024-03-27 11:57:37 | mtr google.com from New York --limit 2
> https://www.jsdelivr.com/globalping?measurement=SX1NBgfDKiabM1vZ
```

### CODE REVIEW

**Comment by jimaek** [comment]
Those flags are probably the most useful options we have, my moving them deeper and effectively removing from the main readme and CLI docs we will make it much harder for people to understand the functionality and to find them.

File: README.md [hunk 1]
```diff
@@ -93,6 +93,7 @@ Measurement Commands:
 Additional Commands:
   completion    Generate the autocompletion script for the specified shell
   help          Help about any command
+  history       Show the history of your measurements
```
Comment (MartinKolarik) [inline README.md:96]: add "in your current session"

**Review by MartinKolarik — COMMENTED** [review]
- [x] if there's no history, the output should be empty, no errors
- [x] the order appears to be reversed; it needs to be old -> new, and the number must match exactly what I'd get when using `ping from @x` 
- [x] ^ also means commands that use `previous` etc don't get their own number in history, see the example in my original issue (between 2 and 3 there's one without number)

**Comment by MartinKolarik** [comment]
Let's also remove the default limit and show all items by default.

**Comment by MartinKolarik** [comment]
Also I think all unix tools use `--head/--tail` instead of `--first/--last` so let's rename that as well.

**Pull request merged**